dockerfiles/che/Dockerfile.centos

```diff
@@ -8,17 +8,16 @@
 #   Dharmit Shah  - Initial implementation
 #   Mario Loriedo - Improvements
 #
-# To build it, run in the current folder after executing build.sh:
-#  `docker build -t registry.centos.org/eclipse/che-server -f Dockerfile.centos .`
+# To build it, in current folder:
+#  cp Dockerfile.centos Dockerfile
+#  ./build.sh
 #
 # To run it:
-#  docker run --net=host \
-#             --name che \
-#             -v /var/run/docker.sock:/var/run/docker.sock \
-#             -v /home/user/che/lib:/home/user/che/lib-copy \
-#             -v /home/user/che/workspaces:/home/user/che/workspaces \
-#             -v /home/user/che/storage:/home/user/che/storage \
-#             registry.centos.org/eclipse/che-server
+#  docker run -d -p 8080:8080 \
+#            --name che \
+#            -v /var/run/docker.sock:/var/run/docker.sock \
+#            -v ~/.che/workspaces:/data \
+#            eclipse/che-server:nightly
```

Comment:
about the first point if you use build script by default it will produce nightly so if you need to run this newly image you'll have to use that version